tern/__main__.py

```diff
@@ -104,6 +104,16 @@ def do_main(args):
             else:
                 run.execute_docker_image(args)
                 logger.debug('Report completed.')
+        elif args.oci_image:
+            # Check if the image is of oci://<image-location>:<image-tag>
+            if not check_oci_image_string(args.oci_image):
+                sys.stderr.write('Error running Tern\n'
+                                 'Please provide docker image '
```

Comment:
is the `--oci-image` command line option designed to handle both a docker image string in `image:tag` and an `oci://<image-location>:<image-tag>` format? If so, we should modify the help string on the parser `add_argument` lines below as that help message only directs users to use `oci://<image-location>:<image-tag>`. If not, this string should be modified to direct users to only provide the oci image location/tag format.

Reply:
I think we should have separate arguments to deal with both types of images and yes I need to change the string message to direct users to only provide the oci image location/tag format.

Reply:
Would it make sense to add a new command line option called `--type, -t` were you can enter `docker` or `oci`?

Reply:
@nishakm I added `-t` option as you mentioned and I think it would make more sense having one option `-i` to define container image and `-t` is type of container image to call respective operations on to the image.